spec/display_spec.rb

```diff
@@ -0,0 +1,96 @@
+require 'require_all'
+require_all 'lib'
+require_relative './fake_console.rb'
+
+describe Display do
```

Comment:
You might want to look into using `let` or a `subject` so you don't have to initialize `Display` the same way for every test. Here's a good explanation https://stackoverflow.com/questions/38437162/whats-the-difference-between-rspecs-subject-and-let-when-should-they-be-used